core/src/main/java/de/flo56958/MineTinker/Modifiers/Types/QuickCharge.java

```diff
@@ -0,0 +1,130 @@
+package de.flo56958.MineTinker.Modifiers.Types;
```

Comment:
Quick Charge is already in Haste implemented. I think it makes more sense to implement this enchant in a modifier that makes Tools faster.

Reply:
Didn't know it was in Haste, I'll remove the QuickCharge modifier.